docs/release-notes/v13.2.1.md

```diff
@@ -0,0 +1,72 @@
+---
+id: v13.2.1
+title: Version 13.2.1 Release Notes
+sidebar_label: v13.2.1
+image: "img/release-note-link-preview.png"
+---
+
+_October 10th, 2024_
+
+:::info
+Mia-Platform Console v13.2.1 is **now in Preview** and will be generally available on October 24th.
+
+Console SaaS users can try out v13.2.1 latest improvements in Preview! Open a Service Request to ask for the creation of a sandbox Company in case you do not have access to any Company.
+
+For self-hosted installations, please read the [following guidelines](#how-to-update-your-console).
+:::
+
+## Console
+
+### Improvements
+
+#### Integration of webhook payload on “Project Created”
+
+Data regarding the template used to create the Project and other informations about it has been added to the “Project Created” event payload. 
+More information can be found on the [respective documentation](/docs/console/company-configuration/events.mdx).
+
+#### Versions details when creating a versioned Marketplace item
+
+We have improved the experience of creating a versioned resource from the Marketplace by adding the possibility to see the detail of each version before instantiating the resource.
+You only have to open the versions selector and click on the "Version details" button: a modal window will open with the details of the selected version.
+
+### Bug Fix
+
+This version addressed the following bugs:
+
+* we fixed a bug that caused the Project Homepage to not show running pods even though they were present;
+* we fixed a bug that caused the environments selector to disappear once you clicked on an extension at the Runtime level;
+* we fixed a bug that was causing the `Tag Created` event to not include `project` data in the payload;
+* we fixed a wrong behavior that prevented the user to correctly save the configuration after performing a merge of configurations.
+
+## Fast Data
+
+### 
+
+### Fast Data Services
+
+#### 

```

Comment:
```suggestion
#### Control Plane Frontend

The new version `0.4.1` of the Control Plane frontend service in Fast Data Control Plane application is available!

##### Improvements

Added the logout button in the navigation header so that it is possible to logout even in case of errors or unauthorized access.



```